tensorflow/lite/python/BUILD

```diff
@@ -240,7 +239,6 @@ py_test(
     srcs = ["convert_test.py"],
     srcs_version = "PY2AND3",
     tags = [
-        "no_cuda",
```

Comment:
@deven-amd are we sure this is passing? I added it last week because it's failing on CI.

Reply:
yup. 

just ran the entire list of "no_cuda" tests (to figure out which ones are passing) and this one was in the passing column :)

```
# //tensorflow/examples/saved_model/integration_tests:saved_model_test     PASSED in 25.1s
# //tensorflow/examples/saved_model/integration_tests:saved_model_test_gpu PASSED in 27.3s
# //tensorflow/lite/python:convert_test                                    PASSED in 5.6s
# //tensorflow/lite/python:lite_test                                       PASSED in 28.6s
# //tensorflow/python/data/kernel_tests:dataset_test                       PASSED in 6.2s
# //tensorflow/python/distribute:mirrored_strategy_test                    PASSED in 16.0s
# //tensorflow/python/distribute:mirrored_strategy_test_gpu                PASSED in 13.8s
# //tensorflow/python/eager:def_function_test                              PASSED in 10.8s
# //tensorflow/python/eager:def_function_test_gpu                          PASSED in 15.8s
# //tensorflow/python/keras:backend_test                                   PASSED in 14.0s
# //tensorflow/python/keras:base_layer_test                                PASSED in 22.0s
# //tensorflow/python/keras:normalization_test                             PASSED in 26.0s
# //tensorflow/python/keras:training_eager_test                            PASSED in 37.2s
# //tensorflow/python/kernel_tests:unique_op_test                          PASSED in 9.2s
# //tensorflow/python/ops/parallel_for:xla_control_flow_ops_test           PASSED in 7.1s
# //tensorflow/python/ops/parallel_for:xla_control_flow_ops_test_gpu       PASSED in 4.6s
# //tensorflow/python/saved_model:save_test                                PASSED in 11.4s



# //tensorflow/lite/python:interpreter_test                                FAILED in 2.4s
# //tensorflow/python/compiler/xla:xla_test                                FAILED in 4.4s
# //tensorflow/python/compiler/xla:xla_test_gpu                            FAILED in 4.1s
# //tensorflow/python/eager:def_function_xla_test_cpu                      FAILED in 4.4s
# //tensorflow/python/eager:def_function_xla_test_gpu                      FAILED in 4.2s
# //tensorflow/python/keras:convolutional_recurrent_test                   FAILED in 1 out of 4 in 29.4s
# //tensorflow/python/kernel_tests:control_flow_ops_py_test                FAILED in 1 out of 16 in 13.8s
# //tensorflow/python/kernel_tests:control_flow_ops_py_test_gpu            FAILED in 1 out of 16 in 9.7s
# //tensorflow/python/training/tracking:util_xla_test_cpu                  FAILED in 4.3s
# //tensorflow/python/training/tracking:util_xla_test_gpu                  FAILED in 4.3s

# Executed 27 out of 27 tests: 17 tests pass and 10 fail locally.
# INFO: Build completed, 10 tests FAILED, 25066 total actions
```